test/01LessonOneTest.js

```diff
@@ -5,30 +5,30 @@ var assert = require('assert');
 
 function returnTrue() {
     // should return true
-
+return true;
 }
```

Comment:
Good to see you're making progress, @kelleymccauley!

I do tend to indent the code inside functions, though, to make it easier to read.

LIke so:

```javascript
function returnTrue() {
    // should return true
    return true;
}
```